end-to-end-test/remote/specs/need-fixing/studyview.spec.js

```diff
@@ -0,0 +1,125 @@
+const assert = require('assert');
```

Comment:
this file was remove (test code was moved to core spec folder)

Reply:
Yes indeed. Existing tests have been updated.